### DISCUSSION TITLE
Dev Home

### DISCUSSION THREAD

**Discussion by MarvinG6547** [ChrisTitusTech/winutil#2978] [Ideas]
It would be really great if a option were to be added that completely removes Dev Home.
I got so angry at Microsoft for installing that on my pc the other night that I went into the windowsapps folder and deleted everything relating to Dev Home and denied the system from writing to that folder, but it's still there and it keeps installing itself.

**Comment by AndreyGasik** [comment]
Use uupdump website, download they script, correct custom list app what you need and create iso without all garbage, after install you can use Microwin for apply tweaks and install what you need.
